plugins/archlinux/README.md

```diff
@@ -106,6 +112,8 @@ upgrades were available. Use `pacman -Que` instead.
 | Alias   | Command                           | Description                                                         |
 |---------|-----------------------------------|---------------------------------------------------------------------|
 | trconf  | `trizen -C`                       | Fix all configuration files with vimdiff                            |
+| trclean | `trizen -Sc`                      | Clean out old and unused caches and packages                        |
+| trclr   | `trizen -Scc`                     | Remove all files from the cache                                     |
```

Comment:
I'm not sure about these two. Documentation is harder to link to, but [here's the man source code](https://github.com/trizen/trizen/blob/fef9dec28b8d155f642c9e1a1b5b5454fb674548/TRIZEN.md#sync-options--s):
> ## SYNC OPTIONS (`-S`)
> usage: trizen {-S --sync} [options] [package(s)]
> 
> Main options:
> 
>   -c, --clean         : clean the cache directory of `trizen` and `pacman`

@harishnkr have you checked what `trizen -Scc` does?

Reply:
yes it works in the same way as the -Scc flag in pacman